locales/it_IT/pcsx2_Main.po

```diff
@@ -731,143 +742,141 @@ msgstr ""
 
 #: pcsx2/gui/ConsoleLogger.cpp:115
 msgid "Save log question"
-msgstr "Conferma salvataggio log"
+msgstr "Conferma salvataggio registro"
 
-#: pcsx2/gui/ConsoleLogger.cpp:389
+#: pcsx2/gui/ConsoleLogger.cpp:390
 msgid "&Small"
 msgstr "Piccolo"
 
-#: pcsx2/gui/ConsoleLogger.cpp:389
+#: pcsx2/gui/ConsoleLogger.cpp:390
 msgid "Fits a lot of log in a microcosmically small area."
-msgstr "Infila un mucchio di log in un'area microcosmicamente piccola."
+msgstr "Infila un mucchio di registro in un'area microcosmicamente piccola."
 
-#: pcsx2/gui/ConsoleLogger.cpp:391
+#: pcsx2/gui/ConsoleLogger.cpp:392
 msgid "&Normal font"
 msgstr "Carattere normale"
 
-#: pcsx2/gui/ConsoleLogger.cpp:391
+#: pcsx2/gui/ConsoleLogger.cpp:392
 msgid "It's what I use (the programmer guy)."
 msgstr "È quello che uso io (il programmatore)."
 
-#: pcsx2/gui/ConsoleLogger.cpp:393
+#: pcsx2/gui/ConsoleLogger.cpp:394
 msgid "&Large"
 msgstr "Grande"
 
-#: pcsx2/gui/ConsoleLogger.cpp:393
+#: pcsx2/gui/ConsoleLogger.cpp:394
 msgid "Its nice and readable."
 msgstr "È bello e leggibile."
 
-#: pcsx2/gui/ConsoleLogger.cpp:395
+#: pcsx2/gui/ConsoleLogger.cpp:396
 msgid "&Huge"
 msgstr "Enorme"
 
-#: pcsx2/gui/ConsoleLogger.cpp:395
+#: pcsx2/gui/ConsoleLogger.cpp:396
 msgid "In case you have a really high res display."
 msgstr "Nel caso tu possieda uno schermo veramente ad alta risoluzione."
 
-#: pcsx2/gui/ConsoleLogger.cpp:399
+#: pcsx2/gui/ConsoleLogger.cpp:400
 msgid "&Light theme"
 msgstr "Tema chiaro"
 
-#: pcsx2/gui/ConsoleLogger.cpp:399
+#: pcsx2/gui/ConsoleLogger.cpp:400
 msgid "Default soft-tone color scheme."
 msgstr "Schema predefinito a colori tenui."
 
-#: pcsx2/gui/ConsoleLogger.cpp:400
+#: pcsx2/gui/ConsoleLogger.cpp:401
 msgid "&Dark theme"
 msgstr "Tema scuro"
 
-#: pcsx2/gui/ConsoleLogger.cpp:400
+#: pcsx2/gui/ConsoleLogger.cpp:401
 msgid ""
 "Classic black color scheme for people who enjoy having text seared into "
 "their optic nerves."
 msgstr ""
 "Schema classico a colore nero, per la gente che si diverte ad avere il testo "
 "marchiato a fuoco nei propri nervi ottici."
 
-#: pcsx2/gui/ConsoleLogger.cpp:407
+#: pcsx2/gui/ConsoleLogger.cpp:408
 msgid "&Save..."
 msgstr "&Salva..."
 
-#: pcsx2/gui/ConsoleLogger.cpp:407
+#: pcsx2/gui/ConsoleLogger.cpp:408
 msgid "Save log contents to file"
-msgstr "Salva i contenuti del log in un file."
+msgstr "Salva i contenuti del registro in un file"
 
-#: pcsx2/gui/ConsoleLogger.cpp:408
+#: pcsx2/gui/ConsoleLogger.cpp:409
 msgid "C&lear"
 msgstr "Pu&lisci"
 
-#: pcsx2/gui/ConsoleLogger.cpp:408
+#: pcsx2/gui/ConsoleLogger.cpp:409
 msgid "Clear the log window contents"
-msgstr "Cancella i contenuti della finestra di log."
+msgstr "Cancella i contenuti della finestra di registro"
 
-#: pcsx2/gui/ConsoleLogger.cpp:410
+#: pcsx2/gui/ConsoleLogger.cpp:411
 msgid "Auto&dock"
-msgstr ""
+msgstr "Finestra a&gganciata"
```

Comment:
Mgh?

Reply:
La "&" indica a quale lettera della tastiera la voce del menu rispondere.
Ad es. l'opzione File->Nuovo è solitamente raggiungibile con Alt+F e poi N .

Se invece il commento si riferisce alla traduzione, non ne sono riuscito a trovare una migliore.
Ho già visto la traduzione "dock"->"aggancio" in altri programmi.
L'opzione permette di trascinare la finestra di log insieme a quella principale di PCSX2.